index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
```

Comment:
There are some files bleeding in the push

Reply:
sorry -.-